Source/Private/ASDisplayNodeInternal.h

```diff
@@ -204,9 +204,9 @@ FOUNDATION_EXPORT NSString * const ASRenderingEngineDidDisplayNodesScheduledBefo
   NSMutableArray<ASDisplayNode *> *_yogaChildren;
 #endif
 #if YOGA_TREE_CONTIGUOUS
-  YGNodeRef _yogaNode;
   __weak ASDisplayNode *_yogaParent;
   ASLayout *_yogaCalculatedLayout;
+  BOOL _yogaLayoutInProgress;
```

Comment:
This replaces the ASHierarchyState options. ASHierarchyState propagation turned out to not be desirable, as the Yoga layout may be used on any individual parent+children combination, or contiguous subtrees separated by non-Yoga nodes too.